gonzo/scripts/base.py

```diff
@@ -6,32 +7,31 @@
 from gonzo.exceptions import ConfigurationError
 from gonzo.scripts import config
 
-from gonzo.scripts.image import create as image_create
-from gonzo.scripts.image import delete as image_delete
+# from gonzo.scripts.image import create as image_create
```

Comment:
can we just delete instead of leaving commented out?

and if unused, let's delete the corresponding modules as well